### DISCUSSION TITLE
How to switch back to monochrome icons/emojis in i3 bar / blocks? Weird colorful font issue!

### DISCUSSION THREAD

**Discussion by therealmaxmoon** [i3/i3#5791] [Q&A]
I really like the i3 blocks status bar with a few colors.

But after installing (full) LaTeX, all icons/emojis are really colorful, now, which I don't like.

How can I go back to the beautiful monichrome icons without uninstalling LaTeX?

In my i3 config in the bar block, I have the following:

`font pango:DejaVu Sans Mono for Powerline, FontAwesome 12`

Thanks a lot for your help.

**Comment by GersonFeDutra** [comment]
Same after running the following script to enable emojis:
```bash
#!/bin/bash
sudo pacman -S noto-fonts noto-fonts-emoji ttf-liberation ttf-droid adobe-source-sans-fonts ttf-dejavu
git clone https://aur.archlinux.org/ttf-ms-fonts.git
cd ttf-ms-fonts
makepkg -si
cd ..
rm -rf ttf-ms-fonts
```
I have the following: `font pango:FontAwesome 10, DejaVu Sans Mono 8, monospace 8`

**Reply by therealmaxmoon** [reply]
I don't understand why this should be a solution, because you install a lot of fonts from the Arch repo, after that you install one font from the AUR repo (I still find AUR questionable, because the last time I checked, it wasn't secure [but this was years ago]) and I somehow don't know how Microsoft fonts (if ms stands for that) could help here?

**Reply by GersonFeDutra** [reply]
The issue is that I had monochrome bar _before_ installing those fonts. I still need these fonts, but I do want the bar as before.

**Reply by therealmaxmoon** [reply]
This issue still exists :( I couldn't find a solution.
